Automattic/wp-calypso
# Remove Unused & Duplicate AdWords Illustration

#### Changes proposed in this Pull Request

As mentioned [here](https://github.com/Automattic/wp-calypso/issues/36850#issuecomment-596816734), unused illustrations should be removed as part of #36850, though they could always be recovered in the future. This is one of them.

#### Testing instructions

Verify there's no uses of this illustration: https://github.com/Automattic/wp-calypso/search?q=%22adWords.svg%22&unscoped_q=%22adWords.svg%22

Note that a duplicate illustration exists here too: https://github.com/Automattic/wp-calypso/blob/master/static/images/upgrades/adwords.svg

Part of #36850
